Cargo.toml

```diff
@@ -1,22 +1,8 @@
-[package]
-name = "cueball"
-version = "0.2.1"
-authors = [
-        "Kelly McLaughlin <kelly.mclaughlin@joyent.com>",
-        "Jon Anderson <jon.anderson@joyent.com>",
-        "Isaac Davis <isaac.davis@joyent.com>"
+[workspace]
+members = [
+    "cueball",
+    "manatee-primary-resolver",
```

Comment:
Would it work to organize the resolver under a `resolvers` subdirectory and use a `connections` subdirectory for the connections? It would just make it a little bit easier to see what was what.

Reply:
Good idea, will do.